src/back/Tile.js

```diff
@@ -31,7 +31,7 @@ Tile.prototype.render = function (project, map, cb) {
     this.setupBounds();
     map.zoomToBox(this.projection.forward([this.minX, this.minY, this.maxX, this.maxY]));
     var im = new mapnik.Image(this.height, this.width);
-    map.render(im, {scale: project.mml.scale}, cb);
+    map.render(im, {scale: project.mml.scale, variables: {zoom: this.z}}, cb);
```

Comment:
I think there is no need given it's already vector data, so scalable consumer side, but I may miss something.
I'll merge as is, we'll see about line 41 is the need arise.